gitlab/tests/conftest.py

```diff
@@ -71,18 +72,53 @@ def dd_environment():
 
 @pytest.fixture()
 def mock_data():
-    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'metrics.txt')
-    with open(f_name, 'r') as f:
-        text_data = f.read()
     with mock.patch(
         'requests.get',
-        return_value=mock.MagicMock(
-            status_code=200, iter_lines=lambda **kwargs: text_data.split("\n"), headers={'Content-Type': "text/plain"}
-        ),
+        side_effect=mocked_requests_get,
     ):
         yield
 
 
+def mocked_requests_get(*args, **kwargs):
+    if args[0].startswith("http://{}:{}/-/readiness".format(HOST, GITLAB_LOCAL_PORT)):
```

Comment:
Could you use sequence unpacking to assign `args[0]` to a meaningful name?

```
x, *rest_args = args  # s/x/something_meaningful
```

Reply:
Good idea, I just added `url = args[0]` as this is the only thing I need there